peps/pep-0696.rst

```diff
@@ -15,8 +15,8 @@ Post-History: `22-Mar-2022 <https://mail.python.org/archives/list/typing-sig@pyt
 Abstract
 --------
 
-This PEP introduces the concept of type defaults for
-``TypeVarLike``\ s (``TypeVar``, ``ParamSpec`` and ``TypeVarTuple``),
+This PEP introduces the concept of type defaults for type parameters
+(and TypeVar, ParamSpec and TypeVarTuple for backwards compatibilty),
```

Comment:
This wording is a bit confusing because PEP 695 still creates TypeVar objects. I would use this:
```suggestion
This PEP introduces the concept of type defaults for type parameters,
including ``TypeVar``, ``ParamSpec``, and ``TypeVarTuple``,
```